zmon-common/src/main/java/org/zalando/zmon/domain/ServiceLevelStatus.java

```diff
@@ -59,21 +65,29 @@ public void fillMessage() {
                 this.message = "SERVICE DEGRADATION: ";
             }
 
-            if (this.queryMaxCheckTier != 3) {
-                this.message += "Metric visualization is currently only available for " + this.checkTiers.get(this.ingestMaxCheckTier) + " ";
+            if (this.queryDistanceHoursLimit == 12 && this.ingestMaxCheckTier == 3 && this.ingestMaxCheckTier == 3) {
+                // level 1
+                this.message += FIRST_LEVEL_WARNING;
+            }
+
+            if (this.queryDistanceHoursLimit == 12 && this.ingestMaxCheckTier == 3 && this.ingestMaxCheckTier == 2) {
```

Comment:
I think there are some copy paste problems in your if conditions, please double check